polynote-frontend/polynote/data/messages.ts

```diff
@@ -61,6 +61,19 @@ export class LoadNotebook extends Message {
     }
 }
 
+export class CloseNotebook extends Message {
```

Comment:
This message now gets sent when you close a notebook tab (so we can unsubscribe you and possibly close the notebook overall)

Reply:
Hm, should we also send this message when you close your browser window?

Reply:
Ah, I missed that part of `removeSubscriber` where we close the notebook when the socket gets closed 😅